include/poller.php

```diff
@@ -164,13 +181,52 @@ function poller_execute($queue) {
 
 		$a->process_id = $old_process_id;
 
+		$duration = number_format(microtime(true) - $stamp, 3);
+
+		logger("Process ".$mypid." - Prio ".$queue["priority"]." - ID ".$queue["id"].": ".$funcname." - done in ".$duration." seconds.");
+
+		if (Config::get("system", "profiler")) {
```

Comment:
here could also be a short comment which describes was the block in the condition does